### ISSUE TITLE
Desktop: System.ConfigurationTests.TypeUtilTests.GetType_NoAssemblyQualifcation failed with "Xunit.Sdk.EqualException"

### ISSUE THREAD

**Issue by Jiayili1** [dotnet/corefx#18596]
Failed test: System.ConfigurationTests.TypeUtilTests.GetType_NoAssemblyQualifcation

Configuration: outerloop_netfx_windows_nt_debug

Detail: https://ci.dot.net/job/dotnet_corefx/job/master/job/outerloop_netfx_windows_nt_debug/23/testReport/System.ConfigurationTests/TypeUtilTests/GetType_NoAssemblyQualifcation_typeString____System_Uri____expectedType__typeof_System_Uri__/

MESSAGE:
~~~
Assert.Equal() Failure
     Expected: typeof(System.Uri)
     Actual: (null)
~~~
STACK TRACE:
~~~
at System.ConfigurationTests.TypeUtilTests.GetType_NoAssemblyQualifcation(String typeString, Type expectedType) 
in D:\j\workspace\outerloop_net---903ddde6\src\System.Configuration.ConfigurationManager\tests\System\Configuration\TypeUtilTests.cs:line 42
~~~

**Comment by Jiayili1** [comment]
This test is disable by https://github.com/dotnet/corefx/issues/19340

**Comment by JeremyKuhne** [comment]
Fixed via #19576
